app/src/main/java/org/wikipedia/util/FeedbackUtil.kt

```diff
@@ -110,10 +110,10 @@ object FeedbackUtil {
         UriUtil.visitInExternalBrowser(context, Uri.parse(context.getString(urlStr)))
     }
 
-    fun composeFeedbackEmail(context: Context, subject: String, body: String = "") {
+    fun composeFeedbackEmail(context: Context, email: String = context.getString(R.string.support_email), subject: String, body: String = "") {
```

Comment:
Since we are using this function for not just the feedback email, can we rename it to `composeEmail`?

Reply:
Done.